### PR TITLE
Introduced the people picker view.

### PR DESCRIPTION
**Pull request by alex-voodoo**
This PR introduces a new view in the People application that allows picking the people based on simple filters.

### CODE REVIEW

File: people/templates/people/picker.html [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    }
+    return result;
+}
+
+function hasAvailablePeople() {
+    for (const option of availableList.options) {
+        if (!option.hidden)
+            return true;
+    }
+    return false;
```
Comment (anarute) [inline people/templates/people/picker.html:83]: Also optional, but to make the logic clearer, I would call the `availablePeopleCount` here:

```suggestion
function hasAvailablePeople() {
    return availablePeopleCount() > 0;
```

**Review by anarute — APPROVED** [review]
Interesting feature!

I'm a bit concerned of how the back end code is mixed in the front end (mainly in the javascript part), and not having a good separation of concerns, but I don't won't to block merging it, we can improve this once we have a better vision of the front end architecture.

As a minor a suggestion I would `let` instead of `var` since it's scoped and also added some nit comments.

File: people/templates/people/picker.html [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    </div>
+    <div class="t-cell people-picker-list">
+        <h2>Picked</h2>
+
+        <select multiple id="picked-list" class="people-picker-list">
+        </select>
+    </div>
+</div>
+
+<script language="JavaScript">
```
[the remark this answers is not in the document]
Comment (alex-voodoo) [reply people/templates/people/picker.html:44]: Good to know!  I have it in other places, too…

File: people/templates/people/picker.html [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+{% endfor %}
+};
+
+function availablePeopleCount() {
+    var result = 0;
+    for (const option of availableList.options) {
+        if (!option.hidden)
+            ++result;
+    }
+    return result;
```
[the remark this answers is not in the document]
Comment (alex-voodoo) [reply people/templates/people/picker.html:75]: This way is definitely less verbose but it is also less efficient?  First it creates an array, then it creates a filtered array, just to return length of that second array.

File: people/templates/people/picker.html (hunk 1 above)
Comment (alex-voodoo) [reply people/templates/people/picker.html:83]: I think that performance-wise it is not worth that.

**Comment by alex-voodoo** [comment]
> Interesting feature!
> 
> I'm a bit concerned of how the back end code is mixed in the front end (mainly in the javascript part), and not having a good separation of concerns, but I don't won't to block merging it, we can improve this once we have a better vision of the front end architecture.
> 
> As a minor a suggestion I would `let` instead of `var` since it's scoped and also added some nit comments.

Yes, let us merge now and improve later, I have some ideas on extending this feature, and I'd like to learn more about better separation (I don't know about best practices how to generate the JS code with Django).

**Pull request merged**